translation/pt/patterns/30-day-warranty.md

```diff
@@ -0,0 +1,76 @@
+## Title
+
+Garantia de 30 dias
+
+## Patlet
+
+Ao aceitar contribuições de fora da sua própria equipe, há uma aversão natural em assumir a responsabilidade pelo código que não foi escrito pela própria equipe. Através da Garantia de 30 dias, a equipe contribuinte concorda em fornecer correções de bugs para a equipe receptora, o que aumentará o nível de confiança entre as duas equipes e torna mais provável que as contribuições sejam aceitas.
+
+## Problema
+
+Uma equipe desenvolve um componente que é utilizado em toda a organização. Essa equipe resiste em aceitar ou rejeita completamente contribuições (pedidos de recursos). Esse comportamento bloqueia o progresso e leva a interrupções frequentes devido a escalonamentos.
+
+## Contexto
+
+- As equipes dependem de outra equipe para aceitar suas contribuições para que um componente produzido pela equipe receptora possa ser utilizado pela equipe contribuinte.
+- A equipe receptora não possui os recursos, conhecimento, permissão e/ou inclinação para escrever o componente/pedido de recurso por si própria.
+
+## Forças
+
+- Existe desconfiança em relação às contribuições devido a um histórico de trapaças: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção.
```

Comment:
Mudaria de "trapaças" para "incrédulo": "Existe desconfiança em relação às contribuições devido a um histórico incrédulo: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção."

Reply:
Olá @DanielleAlmeida, fazendo a adaptação para utilizar o incrédulo, não sei se passa a mensagem adequada para esse tópico, pois "incredulidade" geralmente se refere a uma falta de crença ou dúvida sobre algo, e não necessariamente a atos de enganação ou trapaça.

> Existe desconfiança em relação às contribuições devido a um histórico de incredulidade: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção.

Talvez utilizar como os Japoneses fizeram e utilizar a expressão "devido a um histórico de má conduta" para indicar de forma mais clara que no passado os desenvolvedores entregaram código incompleto.

> Existe desconfiança em relação às contribuições devido a um histórico de má conduta: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção.

Reply:
As an aside: Great to see even more collaboration on the translation 🥳 Thank you @DanielleAlmeida!